server/http_handler_test.go

```diff
@@ -440,7 +440,7 @@ func (ts *HTTPHandlerTestSuite) TestGetTableMVCC(c *C) {
 	info := data.Value.Info
 	c.Assert(info, NotNil)
 	c.Assert(len(info.Writes), Greater, 0)
-	startTs := info.Writes[0].StartTs
+	startTs := info.Writes[2].StartTs
```

Comment:
bug fix?

Reply:
There will be 4 records returned(the nearest 2 are `op_lock` records), using the `op_lock` record `startTS` may fetch the index key, so using the row key as before